browser/src/react-redux/App/LoginPage.react.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import setUser from './User/User.action'
+import axios from 'axios'
+
+const initialState = {
+  email: '',
+  password: '',
+  error: ''
+}
+
+class LoginPage extends Component {
+  state = initialState
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  isSubmitButtonEnabled = () => {
+    return !(
+      this.state.email &&
+      this.state.password
+    )
+  }
+
+  onSubmit = e => {
+    e.preventDefault()
+    const user = {
+      email: this.state.email,
+      password: this.state.password
+    }
+    axios
+      .post('/api/auth/login', user)
+      .then(res => {
+        this.props.setUser(res.data);
+      })
+      .catch(error => {
+        const err = error.response.data.error.message
+        this.setState({ error: err })
+      })
+  }
+  componentDidUpdate(prevProps) {
+    if (this.props.user !== prevProps.user) {
```

Comment:
We want 2 redirect to the homepage for these 2 scenarios:
1) When we successfully login
2) When we are already logged in

Let's simplify this and just put a `<Redirect>` in the render that will redirect the user when a user object exists in our store.